src/main/java/io/jenkins/plugins/opentelemetry/job/MonitoringPipelineListener.java

```diff
@@ -91,6 +116,28 @@ public void onStartNodeStep(@Nonnull StepStartNode stepStartNode, @Nonnull Strin
         }
     }
 
+    @Override
+    public void onAfterStartNodeStep(@Nonnull StepStartNode stepStartNode, @Nonnull String nodeName, @Nonnull WorkflowRun run) {
+        try (Scope ignored = setupContext(run, stepStartNode)) {
+            verifyNotNull(ignored, "%s - No span found for node %s", run, stepStartNode);
+
+            final long allocationTime = calculateAllocationTime(stepStartNode, PipelineNodeUtil.getPreviousNode(stepStartNode));
+            final String label = fetchNodeLabels(PipelineNodeUtil.getPreviousNode(stepStartNode));
+            String spanNodeName = "Node: " + nodeName;
+            Span nodeSpan = getTracer().spanBuilder(spanNodeName)
+                    .setParent(Context.current())
+                    .setAttribute(JenkinsOtelSemanticAttributes.JENKINS_STEP_TYPE, getStepType(stepStartNode.getDescriptor(), "node"))
+                    .setAttribute(JenkinsOtelSemanticAttributes.JENKINS_STEP_ID, stepStartNode.getId())
+                    .setAttribute(JenkinsOtelSemanticAttributes.JENKINS_STEP_NAME, nodeName)
+                    .setAttribute(JenkinsOtelSemanticAttributes.JENKINS_STEP_NODE_LABEL, label)
+                    .setAttribute(JenkinsOtelSemanticAttributes.JENKINS_STEP_NODE_ALLOCATION_TIME, allocationTime)
+                    .startSpan();
+            LOGGER.log(Level.FINE, () -> run.getFullDisplayName() + " - > node(" + nodeName + ") - begin " + OtelUtils.toDebugString(nodeSpan));
+
+            getTracerService().putSpan(run, nodeSpan, stepStartNode);
+        }
+    }
+
```

Comment:
I was thinking that a problem in node allocation was surfaced regarding the trend of the duration of the interval from `onStartNodeStep` and `onAfterStartNodeStep` and that we just had to create a span for this (rather than nesting the 2 spans.

Something like

```
test-pipeline
   |
   |- Phase: Start
   |
   |- Phase: Run
   |   |
   |   |- Node (Linux)
   |       |
   |       |- Allocate Node (Linux) <-- THIS SPAN REPORTS THE ALLOCATION DURATION
   |       |
   |       |- Stage: ze-stage1
   |       |   |
   |       |   |- shell-1
   |
   |- Phase: Finalise
```

```
   test-pipeline
   |
   |- Phase: Start
   |
   |- Phase: Run
   |   |
   |   |- Node: Allocate (Linux)
   |       |
   |       |- Node: Ready (Linux) <-- HARD TO REPORT THE ALLOCATION DURATION
   |           |
   |           |- Stage: ze-stage1
   |           |   |
   |           |   |- shell-1
   |
   |- Phase: Finalise
```

Reply:
+100